nucypher/cli/actions.py

```diff
@@ -317,12 +371,17 @@ def make_cli_character(character_config,
     return CHARACTER
 
 
-def select_stake(stakeholder, emitter) -> Stake:
+def select_stake(stakeholder, emitter, divisible: bool = False) -> Stake:
     stakes = stakeholder.all_stakes
-    active_stakes = sorted((stake for stake in stakes if stake.is_active),
-                           key=lambda some_stake: some_stake.address_index_ordering_key)
-    enumerated_stakes = dict(enumerate(active_stakes))
-    painting.paint_stakes(stakes=active_stakes, emitter=emitter)
+    stakes = sorted((stake for stake in stakes if stake.is_active), key=lambda s: s.address_index_ordering_key)
+    if divisible:
+        emitter.echo("NOTE: Showing divisible stakes only", color='yellow')
+        stakes = list(filter(lambda s: bool(s.value >= stakeholder.economics.minimum_allowed_locked*2), stakes))
```

Comment:
Isn't the type bool anyway? Or is this bool serving a role of a type hint?

Reply:
Yeah, just a style habit.